packages/web-app-files/src/views/spaces/Projects.vue

```diff
@@ -163,35 +162,26 @@ export default defineComponent({
   },
   setup() {
     const store = useStore()
-    const spaces = computed(() => store.getters['Files/activeFiles'] || [])
+    const spaces = computed(
+      () =>
+        store.getters['runtime/spaces/spaces']
+          .filter((s) => isProjectSpaceResource(s))
+          .sort((a, b) => a.name.localeCompare(b.name)) || []
+    )
     const accessToken = useAccessToken({ store })
-    const { graphClient } = useGraphClient()
 
     const loadResourcesTask = useTask(function* (signal, ref) {
       ref.CLEAR_FILES_SEARCHED()
       ref.CLEAR_CURRENT_FILES_LIST()
-
-      const response = yield unref(graphClient).drives.listMyDrives(
-        'name asc',
-        'driveType eq project'
-      )
-      let loadedSpaces = response.data?.value || []
-
-      loadedSpaces = loadedSpaces.map((s) =>
-        buildSpace({ ...s, serverUrl: configurationManager.serverUrl })
-      )
-      ref.LOAD_FILES({ currentFolder: null, files: loadedSpaces })
-    })
-    const areResourcesLoading = computed(() => {
-      return loadResourcesTask.isRunning || !loadResourcesTask.last
```

Comment:
Fyi: IMO there is no need to load spaces again, we already have them loaded in the store. We can re-use them here, which also makes it easier when updating spaces, because then we don't need to update them twice (in store and in active files). Updating the store is sufficient.